branchtesting/src/main/java/branchexample/App.java

```diff
@@ -1,16 +1,23 @@
 package branchexample;
 
+import java.math.BigDecimal;
+
 public class App 
 {
+	public static String vulnerableString = new String();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static vulnerableString" field final ([squid:S1444](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS1444))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=14&issues=AWWLSHYB3Qu1svVwm1_k&open=AWWLSHYB3Qu1svVwm1_k)

[ISSUE_KEY]: AWWLSHYB3Qu1svVwm1_k